lib/pg.utils.js

```diff
@@ -136,6 +137,23 @@ const buildWhere = query => {
   }).join(' AND '), params];
 };
 
+const expand = (object, schema) => {
```

Comment:
This doesn't 'expand' the object, it actually 'compacts' it. I think it'd be better to name it `fitToSchema` or `frameInSchema` etc?